airflow/models/xcom.py

```diff
@@ -182,6 +189,9 @@ def get_many(
 
         ``run_id`` and ``execution_date`` are mutually exclusive.
 
+        This function returns an SQLAlchemy query of full XCom objects. If you just want one stored value then
+        use :meth:`get_one`.
```

Comment:
Perhaps also mention `with_entities()` for people who want a list of values? (If that’s a meaningful use case.)

Reply:
I think a more whole-sale reworking of this to make the serde handled automatically is in order, so I'm going to leave this as is for now.